tests/automation_framework/data/work_order/work_order_submit_dataEncryptionAlgorithm_list_same_algo_twice.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
I see there is already a file "work_order_dataEncryptionAlgorithm_list_same_algo_twice.json" which seems similar, this looks like a duplicate.

Reply:
Verified but did not find any similar file.